larray/core/array.py

```diff
@@ -1048,9 +1048,9 @@ def to_frame(self, fold_last_axis_name=False, dropna=None):
 
         Parameters
         ----------
-        fold_last_axis_name : bool, optional.
-            False by default.
-        dropna : {'any', 'all', None}, optional.
+        fold_last_axis_name : bool, optional
```

Comment:
I wonder if we should not remove this option from the public API and have an internal _to_io_frame function which would do this if necessary before or after calling to_frame. In that case we could remove it from from_frame too. The question is if it can be useful to read an LArray-formatted file using pandas and then converting it to LArray via from_frame. My impression is that even there are cases where users could want to do that **but** those would include some data manipulation that would make that strange axis name useless. Thoughts?

Reply:
by removing, you mean in both `to_frame` and `from_frame`?

Reply:
Yes, **IF** we go through with this. To be clear, the functionality would still need to live somewhere, but would not be exposed as arguments of public API method/functions.